__tests__/OmnichannelChatSDK.node.spec.ts

```diff
@@ -14,9 +14,9 @@ describe('Omnichannel Chat SDK (Node)', () => {
     AWTLogManager.initialize = jest.fn();
 
     const omnichannelConfig = {
-        orgUrl: '',
-        orgId: '',
-        widgetId: ''
+        orgUrl: '[data-org-uri]',
```

Comment:
I'd recommend to change `[data-org-uri]` to `[data-org-url]` to match the attribute in the code snippet `<script></script>`.